packages/repository/README.md

```diff
@@ -168,7 +168,13 @@ coercion. The following types are supported out of box.
 - ArrayType
 - UnionType
 
-# Use Repository
+## Installation
+
+```
+$ npm install --save @loopback/repository
+```
+
+## Basic use
```

Comment:
Would it make sense to keep just the explanation of `Repository` in the Overview (since this is the package for Repository) and move the description of other things (Model, DataSource, Connector, Mixin, etc) elsewhere?